test/TestServerProjects/media_types/readme.md

```diff
@@ -0,0 +1,10 @@
+# media_types
+
+### AutoRest Configuration
+
+> see https://aka.ms/autorest
+
+```yaml
+modelerfour:
+  lenient-model-deduplication: true
```

Comment:
Why are using `lenient-model-deduplication` here? we shouldn't have duplicate models.

Reply:
We have content types with different options, e.g., 
https://github.com/pshao25/autorest.csharp/blob/contentTypes/test/TestServerProjects/media_types/Generated/Models/ContentType1.cs
https://github.com/pshao25/autorest.csharp/blob/contentTypes/test/TestServerProjects/media_types/Generated/Models/ContentType1AutoGenerated.cs
https://github.com/pshao25/autorest.csharp/blob/contentTypes/test/TestServerProjects/media_types/Generated/Models/ContentType1AutoGenerated2.cs
These are added after M4 changes.

Reply:
If I remove `lenient-model-deduplication: true`, I get below error:
>error   | Duplicate object schemas with 'ContentType1' name  detected.

Do you know why M4 is generating duplicate  object schema with 'ContentType1' ?

Reply:
@timotheeguerin could you answer that?

Reply:
M4 will generate the "AutoGenerated" suffix for the duplicated models. But for ContentType we generated, it will be appended by counter, like 1,2,3...

Reply:
After getting updates from m4, now it [generated code](https://github.com/Azure/autorest.csharp/tree/5e8f565e70f9f143f91773b5fce6d66d9b5b3f80/test/TestServerProjects/media_types/Generated/Models) correctly.